### DISCUSSION TITLE
Drag and drop priority not working?

### DISCUSSION THREAD

**Discussion by kmaustral** [marcusolsson/obsidian-projects#595] [Help] [answered]
I recall it was possible before to drag and drop cards in the board view so that it would change their priority. 

I'm finding now with 1.16 that this is not sticking any more. Cards can be dragged across to different lists horizontally, but not by priority vertically. 

Has anyone else found this problem?

**Comment by marcusolsson** [comment, marked as answer]
This was a breaking change in 1.16. You can read more here: https://github.com/marcusolsson/obsidian-projects/pull/584.

**Comment by kmaustral** [comment]
Thanks for letting me know. I've responded in that discussion.
